docs/content/http-api-authorization.md

```diff
@@ -23,40 +24,94 @@ This tutorial requires [Docker Compose](https://docs.docker.com/compose/install/
 
 ## Steps
 
-### 1. Bootstrap the tutorial environment using Docker Compose.
+### 1. Create a policy bundle.
 
-First, create a `docker-compose.yml` file that runs OPA and the demo web server.
+Create a policy that allows users to request their own salary as well as the salary of their direct subordinates.
+
+**example.rego**:
+
+```live:example:module:openable
+package httpapi.authz
+
+# bob is alice's manager, and betty is charlie's.
+subordinates = {"alice": [], "charlie": [], "bob": ["alice"], "betty": ["charlie"]}
+
+default allow = false
+
+# Allow users to get their own salaries.
+allow {
+  some username
+  input.method == "GET"
+  input.path = ["finance", "salary", username]
+  input.user == username
+}
+
+# Allow managers to get their subordinates' salaries.
+allow {
+  some username
+  input.method == "GET"
+  input.path = ["finance", "salary", username]
+  subordinates[input.user][_] == username
+}
+```
+
+Then, build a bundle.
+
+```shell
+opa build example.rego
+```
+
+You should now see a policy bundle (`bundle.tar.gz`) in your working directory.
+
+### 2. Bootstrap the tutorial environment using Docker Compose.
+
+Next, create a `docker-compose.yml` file that runs OPA, a bundle server and the demo web server.
 
 **docker-compose.yml**:
 
 ```yaml
 version: '2'
 services:
   opa:
-    image: openpolicyagent/opa:{{< current_docker_version >}}
+    image: openpolicyagent/opa:v0.28.0-rootless
```

Comment:
💭 Do we want to change this? Isn't this replaced properly when the website is built?

Reply:
It seems to just point to "opa:latest". Probably not a big deal but it feels good to refer to a version that's actually been tested.

Reply:
👉 https://github.com/open-policy-agent/opa/blob/main/docs/website/layouts/shortcodes/current_docker_version.html not quite, it changes with the docs version selected... It may be worth another discussion, but we seem to default to using these shortcodes in the docs 🤔

Reply:
99.99% of all visitors are going to (or should) browse the latest revision of the docs, and if those translate `{{< current_docker_version >}}` to `latest` that's what everyone is going to see. I personally would much rather we use a tested version regardless of docs revision. The only downside I can think of is that the docs will eventually look a little outdated, but on the other hand that will be because they _are_, and that should be updated with a new tested version. 

On the other hand though I don't feel particularly strongly about this, so if you do I'll happily revert.

Reply:
Can't we get the version out of the website/data/releases.yaml file? I definitely do not want to see us hardcoding the version into the docs because that'll inevitably cause issues for new users running through the tutorials. OTOH, relying on `latest` is not ideal, but since these are __tutorials__ with a focus on getting something working as quickly/easily as possible, I think it's acceptable.

Reply:
> Can't we get the version out of the website/data/releases.yaml file?

You mean whenever you look at https://www.openpolicyagent.org/docs/latest/ (pick "0.28.0 (latest)" in the version picker), it should display the actual latest version (0.28.0) instead of "latest"? Should be possible.

Reply:
Alright, will use `latest`.